### ISSUE TITLE
High Stales on Split

### ISSUE THREAD

**Issue by wasarinta** [Lolliedieb/lolMiner-releases#1434]
![image](https://user-images.githubusercontent.com/81989126/160537896-141925a1-02d9-4f57-b1a1-ad1414e59cf4.png)

both wallet/worker submitted the same share.. can you provide new params for maintenance instead splitting... so no matters dual or single mining ..maintenance will affect to main algo only.

thanks

**Comment by jgonzis** [comment]
> ![image](https://user-images.githubusercontent.com/81989126/160537896-141925a1-02d9-4f57-b1a1-ad1414e59cf4.png)
> 
> both wallet/worker submitted the same share.. can you provide new params for maintenance instead splitting... so no matters dual or single mining ..maintenance will affect to main algo only.
> 
> thanks

Thx for the information... that seems more a problem showing stats!! Let's us analyse it

**Comment by jgonzis** [comment]
Have you put same wallet address in both pools?

**Comment by wasarinta** [comment]
no... i put different wallet for my client and fee. As seen below..

Flight sheet / Mining Profile :
![image](https://user-images.githubusercontent.com/81989126/160598162-d8631fb1-ff44-46e9-947c-31f550959491.png)

Coin/Main Wallet (client wallet) :
![image](https://user-images.githubusercontent.com/81989126/160598329-f7e31002-8bcc-4da7-a996-1e3a0891e533.png)

additional params (value/argument for workername are the same w/ my client workername = RINA. i did it to make it easily to find and identify which one is it) :
![image](https://user-images.githubusercontent.com/81989126/160597344-80274c99-81f3-4a06-bdf6-4b5d2fd31877.png)


Dev.. please add params maintenance so we can add profit for our client in dual mining.. thanks dev.

**Comment by RdWing** [comment]
I also agree would like to see this.

**Comment by jgonzis** [comment]
> I also agree would like to see this.
Problem of split appears when mining the maintenance mode or split to the same pool/server. Because some pools don't consider properly
